src/test/java/usecasetesting/AskQuestionUseCaseTest.java

```diff
@@ -0,0 +1,103 @@
+package usecasetesting;
+
+import adapter.controller.ControlContainer;
+import businessrule.gateway.AttorneyGateway;
+import businessrule.gateway.ClientGateway;
+import businessrule.gateway.QuestionGateway;
+import businessrule.inputboundary.QuestionInputBoundary;
+import businessrule.outputboundary.TheQuestionOutputBoundary;
+import businessrule.requestmodel.QuestionRequestModel;
+import businessrule.responsemodel.TheQuestionResponseModel;
+import businessrule.usecase.AskQuestionInteractor;
+
+import businessrule.usecase.PostDisplayFormatter;
+import businessrule.usecase.QuestionDisplayFormatter;
+import driver.database.*;
+import entity.*;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+;
+
+public class AskQuestionUseCaseTest {
+    final static int CLIENT_ID = 21345678;
+    final static int ATTORNEY_ID = 11345678;
+    final static int SECOND_ATTORNEY_ID = 12222222;
+    private QuestionGateway questionGateway;
+    private QuestionFactory questionFactory;
+    private ClientGateway clientGateway;
+    private AttorneyGateway attorneyGateway;
+    private TheQuestionOutputBoundary theQuestionOutputBoundary;
+    private QuestionInputBoundary questionInputBoundary;
+
+    public void setUpAskQuestionUseCase(){
```

Comment:
Based on my experience testing databases, I recommend to use the BeforeAll annotation for setUp methods to ensure they run before tests during runtime.